.changeset/giant-papayas-attend.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/button-group': patch
+'@twilio-paste/core': patch
+---
+
+[Button group] Fix disabled hover state styles
```

Comment:
issue: I don't think you meant to include button group commits in this PR

Reply:
whoops don't know how that happened, I'm going to make a new PR